risedev.yml

```diff
@@ -343,6 +343,18 @@ profile:
       - use: compute-node
       - use: frontend
 
+  meta-1cn-1fe-sqlite:
```

Comment:
This change seem coming from other unrelated pr, I did not change `risedev.yml` and it's the same as main branch.

Reply:
Can you rebase to eliminate them from your PR?